test/unit/testSummary.js

```diff
@@ -0,0 +1,38 @@
+const initSteps = require('app/core/initSteps');
+const assert = require('chai').assert;
+const sinon = require('sinon');
+const when = require('when');
+const services = require('app/components/services');
+const co = require('co');
+
+describe('Summary', () => {
+    const steps = initSteps([__dirname + '/../../app/steps/action/', __dirname + '/../../app/steps/ui']);
+
+    describe('handleGet()', () => {
+
+        let validateFormDataStub;
+
+        beforeEach(() => {
+            validateFormDataStub = sinon.stub(services, 'validateFormData');
+        });
+
+        afterEach(() => {
+            validateFormDataStub.restore();
+        });
+
+        it('ctx.executorsWithOtherNames returns array of execs with other names', (done) => {
```

Comment:
May need a test for when there isn't an array of execs? @douglasrice will certainly love it if you include them 🥇

Reply:
added tests for when exec list is empty and when no exec has an otherName